  main.cpp
main.obj : error LNK2019: "public: __thiscall Circle::Circle(void)" (??0Circle@@QAE@XZ)_main 함수에서 참조되는 확인할 수 없는 외부 기호
main.obj : error LNK2019: "public: __thiscall Circle::~Circle(void)" (??1Circle@@QAE@XZ)_main 함수에서 참조되는 확인할 수 없는 외부 기호
C:\Users\redqo\Desktop\Cpp\Cpp\Debug\chapter11_class.exe : fatal error LNK1120: 2개의 확인할 수 없는 외부 참조입니다.
